### ISSUE TITLE
Make Namespace mandatory?

### ISSUE THREAD

**Issue by udp** [SynBioDex/SBOL-specification#371]
Is there any reason NOT to use a Namespace?

**Comment by jakebeal** [comment]
In principle, one should always use a Namespace, and when you're using a unified triple collection, there is no reason not to.

For practical purposes, however, I think we need to think through what "requiring a namespace" means for purposes of exchanging objects.  If a Namespace is required, then does that mean that when I download a part from the iGEM collection, that I have to download the entire iGEM catalog in the form of the Namespace collection?  That seems like a heavy-weight interaction that we would not want.

Frankly, I am wondering if we got the directionality of Namespace wrong.  Perhaps a TopLevel should point to its Namespace rather than a Namespace pointing to a bunch of TopLevel objects?

**Comment by udp** [comment]
> Frankly, I am wondering if we got the directionality of Namespace wrong. Perhaps a TopLevel should point to its Namespace rather than a Namespace pointing to a bunch of TopLevel objects?

I think you might be right.

**Comment by jakebeal** [comment]
We talked about this further internally at BBN on our SBOL3 session on Monday, and I think the evidence is leaning more and more strongly that Namespace should not be a collection.  Can we do a quick SEP for 3.0.1 to fix this before it gets baked into anybody's tooling?

**Comment by cjmyers** [comment]
Agreed.  I remember being concerned about this when it was proposed.  Reversing the direction seems a good idea to me.

**Comment by jakebeal** [comment]
I've set up an SEP for this change: https://github.com/SynBioDex/SEPs/issues/102
@udp I set you to be the handling editor; please hand off if you think it should be somebody else instead.
